prototype/html.js

```diff
@@ -0,0 +1,96 @@
+/**
+ * HTML semantics description in accessibility taxonomy terms.
+ *
+ * Text selector syntax:
+ *   :role a pseudo class pointing this element has to be accessible;
```

Comment:
the idea of :role was you could do ":role(grid) > :role(row) > td" to match a grid cell. So I thought if :role is used with no role name on 'this' element then it should mean that this element has to have an accessible element for a match. That is used to apply universal ARIA attributes like aria-label which is applicable to any accessible DOM element. Not sure for now how to make it nicer.